packages/core/src/new-api/types/serialized-deployment.ts

```diff
@@ -79,7 +79,7 @@ export interface SerializedNamedContractDeploymentFuture
   contractName: string;
   constructorArgs: SerializedArgumentType[];
   libraries: SerializedLibraries;
-  value: SerializedBigInt;
+  value: SerializedBigInt | SerializedModuleParameterRuntimeValue;
```

Comment:
Does the change in types imply a change in the serializer?

If we added a test for serialization of modules with Runtime values for `value` would it serialize/deserialize?

Reply:
test added and it does appear to serialize/deserialize properly